guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -10,7 +10,16 @@ There are no highlights with Foreman {ProjectVersion}.
 == Upgrade Warnings
 
 // If this section would be empty otherwise, uncomment the following line:
-There are no upgrade warnings with Foreman {ProjectVersion}.
+// There are no upgrade warnings with Foreman {ProjectVersion}.
+
+=== Running Foreman on Ubuntu 20.04 (Focal) is not supported anymore
+
+Foreman supports running on Ubuntu 22.04 LTS (Jammy Jellyfish) since 3.11.
+Running Foreman on Ubuntu 20.04 LTS has been deprecated since 3.12.
+Support for running Foreman on Ubuntu 20.04 LTS has been removed.
+
+Note this is for running Foreman itself.
+Clients will remain supported.
```

Comment:
should this be inside `ifdef::foreman-deb`?

Reply:
I did that with EL8 because I linked to a guide that was only available on a `foreman-el` build. I think it can make sense.

Reply:
Done!